### ISSUE TITLE
Fix keyboard assisted scrubbing when the time-rail is focused

### ISSUE THREAD

**Issue by joncameron** [samvera-labs/ramp#183]
This works only sometimes with audio, and is more consistent with video. Dananji has put in a ticket with Video.js to inquire about getting this resolved.

With the video, could be a conflict with a custom component in Ramp. 
With audio, it looks like there are community issues with Video.js and others are impacted.

**Comment by joncameron** [comment]
We could wait on this until further changes to Ramp, depending.

**Comment by joncameron** [comment]
We'll need to re-test in current Ramp and Avalon and see if it still needs a fix or this can be closed.

**Comment by joncameron** [comment]
No longer seems to be an issue based on my testing; closing.

**Comment by elynema** [comment]
@joncameron Did you test this on avalon-dev? I'm still not seeing the left/up arrow keys scrubbing through the video when any player controls are focused. Maybe you can demonstrate for me if it's working for you?

**Comment by joncameron** [comment]
I tested this on ramp.avalonmediasystem.org. Not working for me either in avalon-dev; I'll create a new issue for for this on the item page.
